include/wasm.h

```diff
@@ -510,7 +510,7 @@ WASM_API_EXTERN const wasm_memory_t* wasm_extern_as_memory_const(const wasm_exte
 WASM_DECLARE_REF(instance)
 
 WASM_API_EXTERN own wasm_instance_t* wasm_instance_new(
-  wasm_store_t*, const wasm_module_t*, const wasm_extern_t* const imports[],
+  wasm_store_t*, const wasm_module_t*, own wasm_extern_vec_t* imports,
```

Comment:
Why did this change from a `const T*` to an `own T*`? If I understand the code here, this function isn't expected to take ownership of the array.

Reply:
You are right! This was because no other places needed non-own vectors before, so I hadn't noticed it and supporting it required extra infrastructure. Fixed now.